### ISSUE TITLE
In simple 

### ISSUE THREAD

**Issue by vahidalvandi** [denpamusic/php-bitapi#60]
Hi
I am php developer I want send bitcoin form self wallet to another. Dothis package cab help me?

**Comment by simwood-simon** [comment]
Yes, but you'll need a bitcoin node for it to make RPC calls to - it does the RPC calls, the node interacts with the network.

**Comment by vahidalvandi** [comment]
This package use in crypto exchange?

**Comment by jarbey** [comment]
No. This package is "only" (and it's great), a wrapper of the native Json-RPC API of a bitcoin node
